mindmaps-core/src/main/java/io/mindmaps/core/implementation/MindmapsTransactionImpl.java

```diff
@@ -44,23 +44,17 @@
     private final ElementFactory elementFactory;
     private final ConceptLog conceptLog;
     private final AbstractMindmapsGraph mindmapsGraph;
-    private boolean batchLoadingEnabled;
     private Graph transaction;
 
     public MindmapsTransactionImpl(AbstractMindmapsGraph mindmapsGraph) {
         this.mindmapsGraph = mindmapsGraph;
         this.transaction = mindmapsGraph.getGraph();
-        this.batchLoadingEnabled = false;
         conceptLog = new ConceptLog();
         elementFactory = new ElementFactory(this);
     }
 
-    public void setBatchLoadingEnabled(boolean batchLoadingEnabled){
-        this.batchLoadingEnabled = batchLoadingEnabled;
-    }
-
     public boolean isBatchLoadingEnabled(){
-        return batchLoadingEnabled;
+        return mindmapsGraph.isBatchLoadingEnabled();
```

Comment:
Its a short hand for when ConceptImpl needs to know if we batch loading. Basically right now in ConceptImpl it goes:

    if(mindmapsTransaction.isBatchloading()){
        //Do stuff
    }

Without this method I would have to make an accessor for the MindmapsGraph in MindmapsTransaction so the above would become:

    if(mindmapsTransaction.getMindmapsGraph().isBatchloading()){
        //Do stuff
    }

What do you think is better ?